### PR TITLE
mongodb setup, userinfo schema

### LINKED ISSUE #16: User Info Database
## User Story
As a potential user, I would like to have a user info database so that my information is saved securely.

### PR DESCRIPTION
**Pull request by leroynle**
I set up MongoDB for the project and designed a database schema for users' information.

### CODE REVIEW

File: frontend/.eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...
..."12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"151wbru",{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"/home/genuineeek/Workspace/Github/7/frontend/src/index.js",[],["23","24"],"/home/genuineeek/Workspace/Github/7/frontend/src/reportWebVitals.js",[],"/home/genuineeek/Workspace/Github/7/frontend/src/App.js",[],"/home/genuineeek/Workspace/Github/7/frontend/src/components/Pages/Settings.js",[],"/home/genuineeek/Workspace/Github/7/frontend/src/components/Pages/Home.js",[],{"ruleId":"25","replacedBy":"26"},{"ruleId":"27","replacedBy":"28"},"no-native-reassign",["29"],"no-negated-in-lhs",["30"],"no-global-assign","no-unsafe-negation"]
```
Comment (ngducn) [inline frontend/.eslintcache:1]: We might not need this cache file on version control, consider creating a .gitignore for this.

**Review by ngducn — DISMISSED** [review]
I think this looks good! Please take a look at this team @nguyenhoaihuy @sonhdang @TonyYounCH @elielieli909

File: backend/backend/settings.py [hunk 2]
```diff
@@ -93,15 +95,16 @@
 
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
-
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'djongo',
```
Comment (ngducn) [inline backend/backend/settings.py:100]: Isn't it supposed to be "django" instead of "djongo"?

File: frontend/src/components/Pages/Settings.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <h3>This is Setting Page</h3>
+            {this.state.infos.map(item => (
+                <div key={item.id}>
+                    <h3>Username: {item.username}</h3>
+                    <h3>Full Name: {item.name}</h3>
+                    <h3>Email: {item.email}</h3>
+                    <h3>Age: {item.age}</h3>
+                    <h3>Height: {item.height}</h3>
+                    <h3>Weight: {item.weight}</h3>
+                </div>
```
Comment (ngducn) [inline frontend/src/components/Pages/Settings.js:31]: Team please help @LeroyS1 to confirm if this list of info is enough for our users.

File: backend/backend/settings.py (hunk 2 above)
Comment (leroynle) [reply backend/backend/settings.py:100]: No, it should be djongo. Djongo is a Object Database Mapper framework to map python object to MongoDB

**Pull request merged**